pkg/util/containers/v2/metrics/collector_ecs_fargate.go

```diff
@@ -0,0 +1,193 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build docker
+// +build docker
+
+package metrics
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/util/cache"
+	"github.com/DataDog/datadog-agent/pkg/util/ecs/metadata"
+	v2 "github.com/DataDog/datadog-agent/pkg/util/ecs/metadata/v2"
```

Comment:
Shouldn't we import just one version?

Reply:
Also, I see that there's a V3 already. Not sure if we should use that one or not.

Reply:
The current utils we have use v2 (used by the process agent mainly). I choose to stick to the same api version to reduce the entropy introduced by this change. Ofc we can re-evaluate this later.